samples/snippets/process_document_sample_v1beta3.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from google.cloud import documentai_v1beta3 as documentai
+
+# [START documentai_process_document]
+
+# TODO(developer): Uncomment these variables before running the sample.
+# project_id= 'YOUR_PROJECT_ID';
+# location = 'YOUR_PROJECT_LOCATION'; // Format is 'us' or 'eu'
+# processor_id = 'YOUR_PROCESSOR_ID'; // Create processor in Cloud Console
+# file_path = '/path/to/local/pdf';
+
+
+def process_document_sample(project_id: str, location: str, processor_id: str, file_path: str):
+    client = documentai.DocumentProcessorServiceClient()
+
+    # The full resource name of the processor, e.g.:
+    # projects/project-id/locations/location/processor/processor-id
+    # You must create new processors in the Cloud Console first
+    name = f'projects/{project_id}/locations/{location}/processors/{processor_id}'
+
+    with open(file_path, 'rb') as image:
+        image_content = image.read()
+
+    # Read the file into memory
+    document = {
+        'content': image_content,
+        'mime_type': 'application/pdf'
+    }
+
+    # Configure the process request
+    request = {
+        'name': name,
+        'document': document
+    }
+
+    result = client.process_document(request=request)
+
+    def get_text(el):
+        """Document AI identifies form fields by their offsets
+        in document text. This function converts offsets
+        to text snippets.
+        """
+        response = ''
+        # If a text segment spans several lines, it will
+        # be stored in different text segments.
+        for segment in el.text_anchor.text_segments:
+            start_index = int(segment.start_index) if segment.start_index in el.text_anchor.text_segments else 0
+            end_index = int(segment.end_index)
+            response += document.text[start_index:end_index]
+        return response
+
+    print('Document processing complete.')
+
+    document = result.document
+
+    page_1 = document.pages[0]
```

Comment:
question why are we only looking at the first page here? Or am I misunderstanding what's going on?

Reply:
Resolved!